.meta/sinks/humio_logs.toml.erb

```diff
@@ -60,6 +60,20 @@ examples = ["http://myhumiohost.com"]
 required = false
 description = "The optional host to send Humio logs to."
 
+[sinks.humio_logs.options.compression]
+type = "string"
+category = "Requests"
+common = true
+default = "none"
+description = """\
+The compression strategy used to compress the encoded event data before \
+outputting.\
```

Comment:
I'd suggest "transmission" rather than "outputting". Ditto for all the others.

Reply:
Thanks. I'm thinking add compression to `_partials/fields`, because it's same words and data used in few files. Will fix it there.